assets/base.css

```diff
@@ -5563,6 +5586,350 @@ deferred-media {
   font-size: 18px;
 }
 
+/* STRUCTURE: Gute-Nacht-Versprechen */
```

Comment:
Wäre das nicht vielleicht besser in einer eigenen CSS-Datei? Das wird ja nicht von jeder Section genutzt, oder?

Reply:
Im `base.css` haben wir einige Anweisungen, die nur von einzelnen Sections genutzt werden. Ursprünglich hatten wir auch das in eigenen Dateien, die wir aber nach und nach alle aufgelöst haben, weil wir Sachen häufiger definiert haben. Grundsätzlich schreiben wir jetzt vieles direkt in die Datei `base.css`, die über für das Todo-Tree-Plugin für VS Code, was Jakob und ich benutzen, strukturiert ist. Dadurch kann ich in der linken Spalte eine Inhaltsangabe haben, in die diese Abschnitte eingetragen werden (auch wenn Todo-Tree eigentlich für andere Zwecke gedacht ist).

Reply:
Ah OK. Verstehe. 💡